cmd/talisman/talisman_upgrade_job.yaml

```diff
@@ -76,5 +67,5 @@ spec:
       containers:
       - name: talisman
         image: portworx/talisman:latest
-        args: ["-operation",  "upgrade", "-newimage", "portworx/px-enterprise:1.3.0-rc3"]
+        args: ["-operation",  "upgrade", "-ocimonimage", "portworx/oci-monitor", "-ocimontag" ,"1.3.0-rc3"]
```

Comment:
Is there a way to override this values from the command line? We should definitely add that.

Reply:
One way is to wrap the yaml into a bash script. 

See https://github.com/portworx/torpedo/blob/master/deployments/deploy-ssh.sh for example

Is that close to what you want?

Reply:
Added helper script `run_px_upgrade.sh` in incremental. This allows settings ocimonimage and ocimontag from command line with a default for ocimonimage. I have not exposed other parameters in the script since other parameters are primarily for internal testing.